BMM.UI.iOS/Application/NewMediaPlayer/AVPlayerItemRepository.cs

```diff
@@ -46,7 +49,10 @@ public async Task AddAndLoad(IMediaTrack mediaTrack)
 
             CancelPreviousDownloadingIfNeeded();
             PrepareSpaceForFile(mediaTrack.TrackMediaFile.Size);
-            var cacheAVPlayerItemLoader = new CacheAVPlayerItemLoader(_mediaRequestHttpHeaders, mediaTrack.GetUniqueKey);
+            var cacheAVPlayerItemLoader = new CacheAVPlayerItemLoader(
```

Comment:
I don't like calling the constructor directly. I would prefer if DI would handle that.
But I'm not sure if it's better to change it in this case.